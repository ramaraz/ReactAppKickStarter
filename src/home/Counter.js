import React from 'react';

const Counter = ({clickCount}) => (
    <div>Count:{clickCount}</div>
)

export default Counter;