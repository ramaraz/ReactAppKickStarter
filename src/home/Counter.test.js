import React from 'react';
import Counter from './Counter';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const counter = renderer
    .create(<Counter clickCount='122' />)
    .toJSON();
  expect(counter).toMatchSnapshot();
});