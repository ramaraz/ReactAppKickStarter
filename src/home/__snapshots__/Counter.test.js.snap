// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  Count:
  122
</div>
`;
